packages/rpc-graphql/src/loaders/program-accounts.ts

```diff
@@ -19,14 +19,7 @@ async function loadProgramAccounts(
     rpc: Rpc<GetProgramAccountsApi>,
     { programAddress, ...config }: ProgramAccountsLoaderArgs,
 ): Promise<ProgramAccountsLoaderValue> {
-    // @ts-expect-error FIX ME: https://github.com/microsoft/TypeScript/issues/43187
-    return await rpc
-        .getProgramAccounts(
-            programAddress,
-            // @ts-expect-error FIX ME: https://github.com/microsoft/TypeScript/issues/43187
-            config,
-        )
-        .send();
+    return await rpc.getProgramAccounts(programAddress, config).send();
```

Comment:
Awesome! Do you know why this is fixed? The TS issue it was referencing is still open and it doesn't look like we've changed anything here

Reply:
I must have thought it was a different problem, but since I wasn't defining the right type, of course it was throwing a type error (ha).